webapp/components/CommentList/CommentList.vue

```diff
@@ -47,6 +47,33 @@ export default {
         return comment.id === updatedComment.id ? updatedComment : comment
       })
     },
+    scrollCommentsIntoView() {
+      if (!window || !document) {
+        return
+      }
+      const container = document.getElementById('comments')
+      if (container) {
+        const top = container.offsetTop
+        window.scroll({
+          top,
+          left: 0,
+          behavior: 'smooth',
+        })
+      }
+    },
+  },
+  watch: {
+    $route(to, from) {
+      // scroll inside the same page
+      if (to.hash === '#comments') {
+        this.scrollCommentsIntoView()
+      }
+    },
+  },
+  mounted() {
+    if (this.$route.hash === '#comments') {
```

Comment:
do you have an example of when this would happen?
if someone sends another user a deep link with `#comments`?

Reply:
@mattwr18 yes, here is an example go to `More info` then to `Comments` (as described in #1273 ). Deep link is the case as well.

Reply:
Thank you, I'll test that out :)